specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ContainerApps.json

```diff
@@ -405,6 +405,56 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/authtoken": {
```

Comment:
>authtoken [](http://example.com/codeflow?start=129&length=9)

Is this a new APi being introduced or are you trying to get the swagger in sync with the service implementation? Irrespective you will have to get a breaking changes approval before ARM can approve the changes. Please remove the "ARMChangesRequested" tag once you have got the breaking changes approval.